libpod/pod.go

```diff
@@ -302,29 +301,6 @@ func (p *Pod) CgroupPath() (string, error) {
 	if p.state.InfraContainerID == "" {
 		return "", errors.Wrap(define.ErrNoSuchCtr, "pod has no infra container")
```

Comment:
Should this return an error?

If so, can you delete line 298-300 since we already return p.state.CgroupPath below.

Reply:
yeah that should be an error. I removed the extra code, thanks for catching that.